data_steward/cdr_cleaner/cleaning_rules/repopulate_person_post_deid.py

```diff
@@ -206,13 +283,14 @@ def get_repopulate_person_post_deid_queries(project_id,
 
```

Comment:
if using new `pipeline_logging` you can configure here, something like:

```
pipeline_logging.configure(level=logging.DEBUG, add_console_handler=ARGS.console_log)
```

Reply:
I used logging as all the cleaning rules are using this. I have no specific preference to use one or the other I can update this if this is required to be updated.

Reply:
No need to update now, as we discussed I'll bring up in eng sync!